azext_edge/edge/params.py

```diff
@@ -93,7 +93,7 @@ def load_iotops_arguments(self, _):
         context.argument(
             "ops_service",
             options_list=["--ops-service", "--svc"],
-            choices=CaseInsensitiveList(["mq", "dataprocessor", "lnm"]),
+            choices=CaseInsensitiveList(["mq", "dataprocessor", "lnm", "opcua"]),
```

Comment:
We should add the opcua broker resource_kinds below right ?

Reply:
updated thanks!